### PR TITLE
kubernetes v2 docs

### PR DESCRIPTION
**Pull request by jrhizor**

### CODE REVIEW

**Comment by jrhizor** [comment]
A lot of the "operator guide" stuff actually isn't easy to port over to Kubernetes. I think we'll mostly omit those for now since they are mostly useful for internal use.

We should add upgrade docs for Kube, too.

File: docs/contributing-to-airbyte/developing-on-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+We are [working to improve this process](https://github.com/airbytehq/airbyte/issues/4225).
+
+## Completely resetting a local cluster
+
+If you are in a dev environment on a local cluster only running Airbyte and want to start completely from scratch, you can use the following command to destroy everything on the cluster:
+
+```bash
+# BE CAREFUL, THIS COMMAND DELETES ALL RESOURCES, EVEN NON-AIRBYTE ONES!
+kubectl delete "$(kubectl api-resources --namespaced=true --verbs=delete -o name | tr "\n" "," | sed -e 's/,$//')" --all
```
Comment (davinchia) [inline docs/contributing-to-airbyte/developing-on-kubernetes.md:31]: would it be easier to use kubectl delete -k <path to your overlay>?

**Comment by jrhizor** [comment]
@avaidyanatha should we add https://docs.google.com/presentation/d/1vSXy7GgqYjspz4yacHDuqInoxWM4aInuJ_3XhUBTn0U/edit#slide=id.p somewhere for external users or just keep it near our internal architecture docs?

**Comment by avaidyanatha** [comment]
> @avaidyanatha should we add https://docs.google.com/presentation/d/1vSXy7GgqYjspz4yacHDuqInoxWM4aInuJ_3XhUBTn0U/edit#slide=id.p somewhere for external users or just keep it near our internal architecture docs?

We should definitely add this in a public place, this could be important info for debugging.

**Review by avaidyanatha — APPROVED** [review]
Looks awesome, thanks @jrhizor!

**Pull request merged**